ddtrace/mocktracer/mockspan.go

```diff
@@ -92,9 +96,9 @@ func newSpan(t *mocktracer, operationName string, cfg *ddtrace.StartSpanConfig)
 
 type mockspan struct {
 	sync.RWMutex // guards below fields
-	name       string
-	tags       map[string]interface{}
-	finishTime time.Time
+	name         string
+	tags         map[string]interface{}
+	finishTime   time.Time
```

Comment:
formatting is strange

Reply:
as long as linter is not complaining, do we care? :) It is auto-formatted by gopls or gofmt, not sure which one.

Reply:
fixed, sort of.

Reply:
probably `gopls` issue,  maybe you should restart `gopls`, I do it from time to time 😉 
our linter is not checking the formatting. we should at least add `- gofmt` here: https://github.com/signalfx/signalfx-go-tracing/blob/main/.golangci.yml#L27

Reply:
Formatted ddtrace and sarama with gofmt. will have additional formatting changes.

Reply:
resolving 👍